clue/blueprint/cloudify-dev/operations/git.py

```diff
@@ -103,6 +104,15 @@ def checkout(self, branch):
         versions_prefix = '::'
         default_branch = self.branch
         active_feature = self.active_feature
+
+        if '*' in branch:
+            branch = branch.replace('.', '\.')
+            branch_name = self._get_branch_name_from_regex(branch)
+            if branch_name:
```

Comment:
It's nicer to do:
` branch = str(branch_name) if branch_name else default_branch`